### PR TITLE
[LOPS-1927] Add documentation for autopilot composer error

### PR DESCRIPTION
**Pull request by alynnferry**
Closes #

## Summary

**[Troubleshoot Autopilot Error Messages](https://docs.pantheon.io/guides/autopilot/troubleshoot-autopilot/)** - Add troubleshooting for autopilot composer error.

## Effect

The following changes are already committed:

* https://github.com/pantheon-systems/dashboard/pull/3008

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by scottbuscemi** [comment]
@namespacebrian Could you review?

@jazzsequence Can you please merge after BW has reviewed?

**Comment by scottbuscemi** [comment]
Or @stovak

**Comment by jazzsequence** [comment]
@stovak or @namespacebrian can one of you give this a look over so I can click the button?

File: source/content/guides/autopilot/07-troubleshoot.md [hunk 1]
```diff
@@ -367,6 +367,20 @@ Site is running a framework that is not supported. Supported frameworks are:
 
 </Accordion>
 
+## Composer configuration Not Supported
+
+<Accordion title="We could not run Autopilot because the site is running an unsupported framework." id="misconfigured-composer" icon="info-sign">
+
+### Diagnosis
+
+A [Wordpress Composer Managed](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) site which is not using Composer is not supported by Autopilot.
```
Comment (namespacebrian) [inline source/content/guides/autopilot/07-troubleshoot.md:376]: @alynnferry just to confirm.. this is about wordpress-composer-managed sites that aren't using composer _at all_?  Or is it that they aren't using _Integrated Composer_ specifically?

**Comment by namespacebrian** [comment]
Are the failing checks an issue?  linkchecker whatnot and "Welcome / run"?

File: source/content/guides/autopilot/07-troubleshoot.md (hunk 1 above)
Comment (alynnferry) [reply source/content/guides/autopilot/07-troubleshoot.md:376]: I believe it is wordpress-composer-managed sites that are not using Integrated Composer. @danilarff86 Can you confirm?

Comment (scottbuscemi) [reply source/content/guides/autopilot/07-troubleshoot.md:376]: Capitalize P please :) WordPress

Comment (jazzsequence) [reply source/content/guides/autopilot/07-troubleshoot.md:376]:
```suggestion
A [WordPress Composer Managed](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) site which is not using Composer is not supported by Autopilot.
```

File: source/content/guides/autopilot/07-troubleshoot.md [hunk 2]
```diff
@@ -367,6 +367,20 @@ Site is running a framework that is not supported. Supported frameworks are:
...
+
+<Accordion title="We could not run Autopilot because the site is running an unsupported framework." id="misconfigured-composer" icon="info-sign">
+
+### Diagnosis
+
+A [WordPress Composer Managed](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) site which is not using Composer is not supported by Autopilot.
+
+### Solution
+
+Ensure that your [Wordpress Composer Managed](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) site is properly configured. For additional help [contact Support](/guides/support/contact-support/) for assistance.
```
Comment (jazzsequence) [inline source/content/guides/autopilot/07-troubleshoot.md:380]:
```suggestion
Ensure that your [WordPress Composer Managed](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) site is properly configured. For additional help [contact Support](/guides/support/contact-support/) for assistance.
```

**Review by jazzsequence — APPROVED** [review]

**Pull request merged**